src/lambda/commands/deploySamApplication.ts

```diff
@@ -319,3 +325,43 @@ function getDefaultWindowFunctions(): WindowFunctions {
         showInformationMessage: vscode.window.showInformationMessage,
     }
 }
+
+function notifyDuringLongDeployment(deployParameters: DeploySamApplicationParameters): NodeJS.Timeout {
+    return setTimeout(async () => {
+        const cfnClient = new DefaultCloudFormationClient(deployParameters.region)
+        const stack = (await cfnClient.describeStacks(deployParameters.destinationStackName)).Stacks
+        if (stack !== undefined) {
+            const helpLink = localize('AWS.samcli.deploy.stuckStackHelpLink', 'View docs')
+            const awsConsole = localize(
+                'AWS.samcli.deploy.goToCfnDashboard',
+                'Go to {0} console',
```

Comment:
"View Status" or "View Stack" or something like that. "Console" is not a concept we normally refer to. Isn't the purpose here to check the stack status? So the button should be an action with that kind of purpose.

```suggestion
                'View Stack',
```